packages/webapp/src/components/CardWithStatus/TaskCard/TaskCard.jsx

```diff
@@ -67,6 +67,14 @@ export const PureTaskCard = ({
     onClickCompleteOrDueDate?.();
   };
 
+  const isAssigneeInactive = assignee?.status === 'Inactive';
+  let assigneeName = '';
+  if (assignee !== null) {
+    const lastName =
+      assignee.last_name.length > 0 ? assignee.last_name.toUpperCase().charAt(0) + '.' : '';
+    assigneeName = `${assignee.first_name} ${lastName}`;
+  }
+
```

Comment:
for this one I left the `lastName`  in a separate constant just to make it legible as there's a bit of logic to present the last name in a "capital first letter and dot" format (not sure how to call it 😂 ). Also is possible that a username doesn't have a last name, so I left a validation for that.